ci/test-stable.sh

```diff
@@ -46,6 +46,16 @@ for program in programs/native/*; do
   )
 done
 
+# Run program/native/bpf_loader's test and bench with bpf_c feature
+(
+  set -x
+  cd "programs/native/bpf_loader"
+  echo --- program/native/bpf_loader test --features=bpf_c
+  cargo test --verbose --features="bpf_c"
+  echo --- program/native/bpf_loader bench --features=bpf_c
+  cargo +nightly bench --verbose --features="bpf_c" -- --nocapture
```

Comment:
hmm, maybe we should move this to `test-nightly` and keep `+nightly` out of the stable rust docker image

Reply:
I think its awfully large so nightly is probably better.  Also, bench running nightly is probably fine

Reply:
cool yeah let's do that then

Reply:
@mvines, Done and changes uploaded